src/component/feature/FeatureView/FeatureSettings/FeatureSettings.tsx

```diff
@@ -4,52 +4,49 @@ import { useStyles } from './FeatureSettings.styles';
 import { List, ListItem } from '@material-ui/core';
 import ConditionallyRender from '../../../common/ConditionallyRender';
 import FeatureSettingsProject from './FeatureSettingsProject/FeatureSettingsProject';
-import { useHistory, useParams } from 'react-router-dom';
-import { Edit } from '@material-ui/icons';
-import PermissionIconButton from '../../../common/PermissionIconButton/PermissionIconButton';
-import { UPDATE_FEATURE } from '../../../providers/AccessProvider/permissions';
+import { useParams } from 'react-router-dom';
 import { IFeatureViewParams } from '../../../../interfaces/params';
+import { FeatureSettingsInformation } from './FeatureSettingsInformation/FeatureSettingsInformation';
 
+const METADATA = 'metadata';
 const PROJECT = 'project';
 
 export const FeatureSettings = () => {
     const styles = useStyles();
-    const history = useHistory();
     const { projectId, featureId } = useParams<IFeatureViewParams>();
-
-    const [settings, setSettings] = useState(PROJECT);
-
-    const handleEdit = () =>
-        history.push(`/projects/${projectId}/features/${featureId}/edit`);
+    const [settings, setSettings] = useState(METADATA);
 
     return (
         <PageContent headerContent="Settings" bodyClass={styles.bodyContainer}>
             <div className={styles.innerContainer}>
                 <div className={styles.listContainer}>
                     <List>
+                        <ListItem
```

Comment:
These should be `<li>` elements, somehow. (All the children for `<ul>` from `List` should be `<li>`s).